### ISSUE TITLE
visiting a Permalink posted on facebook does not work

### ISSUE THREAD

**Issue by zalexua** [tyrasd/overpass-turbo#420]
Try to post for example https://overpass-turbo.eu/s/Iga on facebook.
You will see that FB will add something like ?fbclid=IwAR1ldgip5Kb6ZoWQzc  to real URL.
When you click it, it will redirect to a "Not Found" page with URL https://overpass-turbo.eu/null

Suggestion is to ignore any URL params (like "fbclid") for Permalinks.

**Comment by CloCkWeRX** [comment]
A bit of a pain, particularly when sharing adhoc queries with non technical groups who are organising through facebook (ie: bike user groups)

**Comment by DaveF63** [comment]
Indeed, it is a bit of a pain. Is the a recent problem, as I'm sure I've posted share urls to FB previously?

**Comment by leighghunt** [comment]
Still live - I just ran into this and thought it was me - but it actually looks like any parameter argument will cause the issue.

E.g. https://overpass-turbo.eu/s/QFI works fine, but https://overpass-turbo.eu/s/QFI?foo=bar ends up going to https://overpass-turbo.eu/null

Unfortunately, this means we're unable to share these links on many social media platforms, which is a shame as it's a great way of sharing information.

**Comment by leighghunt** [comment]
Hi @tyrasd, I had a quick crack at trying to reproduce this issue on a local copy of the code to try and fix it, but couldn't figure out where the `/s` url parameter is handled.

I can see the outbound call using the `short_url_service` setting, just not how it is changed back to a normal URL on the way back in.

Is it possible that the URL un-shortening is happening externally to the code, or in another branch somewhere?

**Comment by DaveF63** [comment]
A work-a-round solution that works for me:
Copy the permalink.
Paste it into an 'ExpandURL' site, such as https://www.expandurl.net/expand

The above is required if MapCSS is used as none of the Export option support it..

Then paste that output into a 'shortenURL' site such as http://tinyurl.com
CopyPaste that into FB.
